### ISSUE TITLE
Cannot access concept card hyper links during talkback

### ISSUE THREAD

**Issue by JishnuGoyal** [oppia/oppia-android#4740]
**Describe the bug**

While using talkback, the hyperlinks inside concept cards are not accessible.

**To Reproduce**
1. Go to Place values topic
2. Click on revision tab
3. Select naming numbers
4. Hyperlink "concept card" cannot be accessed during talkback.

**Expected behavior**
Users should be able to click and navigate to concept card hyperlinks

**Demonstration**
If applicable, add screenshots or videos to help explain your problem.

**Environment**
Device/emulator being used:
Android or SDK version (e.g. Android 5 or SDK 21): Android 12
App version (you can get this through system app settings or via the admin controls menu in-app): 0.10-beta-5e64fae55e

**Additional context**

**Comment by BenHenning** [comment]
@KolliAnitha are you aware of this issue? I'm wondering if it's just a case of @JishnuGoyal not using the swipe up + right flow to bring up the link context menu.

Given that, I don't think we should block the release on this.

**Comment by KolliAnitha** [comment]
@BenHenning, it is working fine when we use the link context menu.

**Comment by BenHenning** [comment]
Thanks for confirming @KolliAnitha! Going ahead and closing this, then.
